preview/galleon-local/src/main/resources/modules/system/layers/base/jakarta/persistence/api/main/module.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright The WildFly Authors
+  ~ SPDX-License-Identifier: Apache-2.0
+  -->
+
+<module xmlns="urn:jboss:module:1.9" name="jakarta.persistence.api">
+
+    <resources>
+        <artifact name="${jakarta.persistence:jakarta.persistence-api}"/>
+    </resources>
+
+    <dependencies>
+        <!-- java.instrument export due to jdeps result -->
+        <module name="java.instrument" export="true"/>
```

Comment:
Why do we export this?

Reply:
Good question on this one.  From what I recall I think the `javassist` bytecode enhancement library [needed it](https://docs.oracle.com/javase/8/docs/api/java/lang/instrument/Instrumentation.html) for application enhancement.  I'm not sure if we need it still...